### ISSUE TITLE
Error on adding TinyMCE to installed apps: AttributeError: 'NoneType' object has no attribute 'endswith'

### ISSUE THREAD

**Issue by Eloque** [aljosa/django-tinymce#106]
Installed on a fresh base, python 2.7.6, pip installed django-tinymce.
Added tinymce to apps, but after running this error appears;

```
>> python manage.py runserver
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/e/Documents/workspace/webshop/lib/python2.7/site-packages/django/core/management/__init__.py", line 385, in execute_from_command_line
    utility.execute()
  File "/Users/e/Documents/workspace/webshop/lib/python2.7/site-packages/django/core/management/__init__.py", line 354, in execute
    django.setup()
  File "/Users/e/Documents/workspace/webshop/lib/python2.7/site-packages/django/__init__.py", line 21, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/Users/e/Documents/workspace/webshop/lib/python2.7/site-packages/django/apps/registry.py", line 108, in populate
    app_config.import_models(all_models)
  File "/Users/e/Documents/workspace/webshop/lib/python2.7/site-packages/django/apps/config.py", line 197, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/Users/e/Documents/workspace/webshop/lib/python2.7/site-packages/tinymce/models.py", line 6, in <module>
    from tinymce import widgets as tinymce_widgets
  File "/Users/e/Documents/workspace/webshop/lib/python2.7/site-packages/tinymce/widgets.py", line 10, in <module>
    import tinymce.settings
  File "/Users/e/Documents/workspace/webshop/lib/python2.7/site-packages/tinymce/settings.py", line 16, in <module>
    JS_ROOT = getattr(settings, 'TINYMCE_JS_ROOT',os.path.join(settings.STATIC_ROOT, 'tiny_mce'))
  File "/Users/e/Documents/workspace/webshop/bin/../lib/python2.7/posixpath.py", line 77, in join
    elif path == '' or path.endswith('/'):
AttributeError: 'NoneType' object has no attribute 'endswith'
```

And my static setting;

```
STATIC_URL = '/static/'

STATICFILES_DIRS = (
    os.path.join(BASE_DIR, "static"),
    '/webapps/scaffold-test/app/Scaffold/static/',
)
```

**Comment by stilobique** [comment]
Thanks you, add this info into the doc !

**Comment by Natim** [comment]
Can you check that this was fixed with: a362a86

**Comment by roammi** [comment]
thk you!
